docs/apis/packaging-models.md

```diff
@@ -5,7 +5,7 @@
 Zip the exported estimator output in your checkpoint directory, e.g.
 
 ```text
-$ ls export/estimator
+$ ls export/estimator/1560263597/
```

Comment:
What is the significance of this number? 1560263597

Reply:
serving version or something